.github/workflows/release.yml

```diff
@@ -104,17 +108,41 @@ jobs:
       -
         name: "Checkout Che Dashboard source code"
         uses: actions/checkout@v2
+
+      -
+        name: Check existing tags
+        run: |
+          set +e
+          RECREATE_TAGS=${{ github.event.inputs.forceRecreateTags }}
```

Comment:
We have pretty much copy pasting. It may be a good time to introduce out github action. How sure how easy/difficult it's to implement.
Maybe we may just encapsulate release script in dockerimage.